docs/envvars.md

```diff
@@ -188,6 +188,13 @@ line of defense" for dev/stage URLs that are public.
 Usually the `server` module is started with `foreman` (the `nf` command)
 and this is the default port.
 
+### `SERVER_WS_PORT`
+
+**Default: `8080`**
+
+This is the default port for the WebSocket server, which is started by default,
+when you run `yarn start`.
```

Comment:
Default or not, I think you can simplify the sentence to:
```
This is the port for the WebSocket server, which is started when you run `yarn start`.
```

Reply:
Done. :-)